src/cowboy_swagger_redirect_handler.erl

```diff
@@ -36,4 +36,5 @@ previously_existed(Req, State) ->
 -spec moved_permanently(Req :: cowboy_req:req(), State :: state()) ->
                            {{true, iodata()}, cowboy_req:req(), state()}.
 moved_permanently(Req, State) ->
-    {{true, "/api-docs/index.html"}, Req, State}.
+    SwaggerPath = application:get_env(cowboy_swagger, path, "/api-docs"),
+    {{true, SwaggerPath ++ "/index.html"}, Req, State}.
```

Comment:
Ooooooh! So this is alredy generating issues in our production systems… nobody relized that because everybody just clicked a link that arleady contained the `index.html` part… HA! 🤦🏻 

Now I see why you wanted to use `path` instead of the full url.
I don't think those 2 things should be necessarily related, @vkatsuba …
I would allow the user to configure the swagger base path (which I would call `base_path` or `root_path`, instead of `path`) on a parameter, and the swagger.json location (which I would call `swagger_json_url` or `swagger_url`) with another parameter.

Reply:
Yes... make sense... So, now I suppose the code should be prepared as well for use at least `path` config. The `application:get_env/3` should be able to get config from top project where it used. Still need to do end to end tests but we mostly there... So, we can prepare different variables now for use them in URL.

Reply:
Ooooh… right… that's an issue. Have you tested your changes on another project using this one as a dependency?
I'm pretty sure it won't work as we expect… because the configuration for the `cowboy_swagger` app can be in _whatever path_… and thus it may not be visible to your new script.

Reply:
I'll write an inline comment in the proper place…

Reply:
Yes... end to end testing will be need to do...